authorization-service/src/functions/basicAuthorizer/handler.ts

```diff
@@ -0,0 +1,70 @@
+import { APIGatewayAuthorizerResult, APIGatewayEvent } from 'aws-lambda';
+import { middyfy } from '~/libs/lambda';
+
+enum PolicyEffects {
+    ALLOW = 'allow',
+    DENY = 'deny',
+}
+
+const generatePolicy = (
+    principalId: string,
+    resource: string,
+    effect = PolicyEffects['ALLOW'],
+): APIGatewayAuthorizerResult => ({
+    principalId,
+    policyDocument: {
+        Version: '2012-10-17',
+        Statement: [
+            {
+                Action: 'execute-api:Invoke',
+                Effect: effect,
+                Resource: resource,
+            },
+        ],
+    },
+});
+
+const basicAuthorizerHandler = async (
+    event: APIGatewayEvent,
+): Promise<APIGatewayAuthorizerResult> => {
+    console.log('Lambda start: basicAuthorizerHandler');
+
+    const {
+        // @ts-ignore
+        authorizationToken,
+        // @ts-ignore
+        methodArn,
+        // @ts-ignore
+        type: eventType,
+    } = event;
+
+    if (eventType !== 'REQUEST') {
+        return generatePolicy(
+            authorizationToken,
+            methodArn,
+            PolicyEffects['DENY'],
+        );
+    }
+
+    try {
+        const buff = Buffer.from(authorizationToken, 'base64');
```

Comment:
In authorizationToken variable you receive the string 'Basic Y2FsbG1ldmxhZGlrOlRFU1RfUEFTU1dPUkQ='
When you try to decode encoded user name and password you have strange string in userName variable due to containing word "Basic". So the lambda returns "access denied" any way. You should cut of the "Basic" before decoding.